src/js/controller/controller.js

```diff
@@ -188,6 +188,7 @@ Object.assign(Controller.prototype, {
                     if (allowPreload && position && duration > 0 && position >= duration - BACKGROUND_LOAD_OFFSET) {
                         mediaModel.off('change:position', onPosition, this);
                         _programController.backgroundLoad(item);
+                        _this.trigger(PRELOAD, { index, item });
```

Comment:
This matches the "playlistItem" event format correct?

Reply:
Maybe this should be triggered in `_programController.backgroundLoad`. I believe this controller forwards all program events. This way we'll always get a "preload" event when `backgroundLoad` is called successfully.